src/views/view_game.py

```diff
@@ -43,6 +48,21 @@ def init_window(self):
         dirty = all_sprites.draw(self.window)
         pygame.display.update(dirty)
 
+    def init_game_btns(self):
+        # Display game buttons area
+        cfg_btns = self.view_config['window']['game_buttons']
+        pygame.draw.rect(self.window, cfg_btns['color'],
+                         (cfg_btns['x'], cfg_btns['y'],
+                          cfg_btns['width'], cfg_btns['height']))
```

Comment:
This is gonna be so much better when we can do cfg_btns = game_view_config.game_buttons, cfg_btns.x, cfg_btns.width...

Reply:
YEAAAAAAH ! I'm waiting for your PR to change it